libs/shared/ui/src/lib/components/avatar/avatar.tsx

```diff
@@ -113,12 +118,13 @@ const AvatarImpl = forwardRef<AvatarImplElement, AvatarImplProps>(function Avata
         //{status === 'idle' || status === 'loading' ? <span className="rt-AvatarFallback" /> : null}
       }
       {status === 'error' ? (
-        <AvatarPrimitive.Fallback className={avatarFallbackVariants({ size, border })}>
+        <AvatarPrimitive.Fallback className={avatarFallbackVariants({ size, border, radius })}>
           {fallback}
         </AvatarPrimitive.Fallback>
       ) : null}
       <AvatarPrimitive.Image
         ref={forwardedRef}
+        className={avatarFallbackVariants({ size, border, radius })}
```

Comment:
It makes more sense to rename `avatarFallbackVariants` to `avatarImplVariants` if it's used in both Fallback and Image.

Reply:
Added thanks  https://github.com/Qovery/console/pull/1588/commits/ce94255e57209df4ceaebc708a3383848a9da3d5